darkTheme.ts

```diff
@@ -1,41 +1,339 @@
-import { createTheme } from '@mui/material/styles';
+import { createTheme, darken } from '@mui/material/styles';
 
-import { themeMuiButtonRoot, themeTypography } from './lightTheme';
+import { themeTypography } from './lightTheme';
+
+const color1 = '#a3a4ad';
+const color2 = '#e9e9e9';
+const color3 = '#141629';
+const color4 = '#2e8540';
+const color5 = '#eb5757';
+
+//const color6 = '#f3f3f3';
```

Comment:
Is it possible to remove that commented-out piece of code?

Reply:
'+